### DISCUSSION TITLE
希望可以增加回复可见功能

### DISCUSSION THREAD

**Discussion by txbk** [halo-dev/halo#4847] [Ideas]
### 你当前使用的版本

2.10.0

### 描述一下此特性

希望可以增加回复可见功能，这个功能对于博客互动很重要呀，可以极大提高热度

**Comment by ruibaby** [comment]
这种功能不建议在 Core 中提供，建议通过插件实现。

当前应用市场也有提供类似功能的插件 <https://www.halo.run/store/apps/app-IzWXo>，如果将此功能放在这个插件里面或许是个不错的选择。

/area plugin
